metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/ExecutionAggregator.java

```diff
@@ -12,27 +11,29 @@
 
 /**
  * An interface defining the methods needed to aggregate RunExecution's
+ * @param <T> The type of execution, example: RunExecution or ValidationExecution
  * @param <M> The aggregated metric from Metrics
  * @param <E> The execution metric from RunExecution
  */
-public interface RunExecutionAggregator<M, E> {
+public interface ExecutionAggregator<T, M, E> {
```

Comment:
Would there be a benefit to bounding any of the generic types, here or elsewhere?  (I don't know the answer.)

Reply:
I bound T because it has to be a type of `Execution`, but the others are not boundable because they don't share a common type